kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Constants.java

```diff
@@ -131,10 +131,6 @@ public interface Constants {
      */
     String DEPLOYMENT = "Deployment";
 
-    /**
-     * Strimzi kafka image url in quay
-     */
-    String STRIMZI_KAFKA_IMAGE = "quay.io/strimzi/kafka:latest-kafka-" + Environment.KAFKA_VERSION;
```

Comment:
Do we not need this?

Reply:
No, we changed the way to install strimzi using a helm chart, but we forgot to remove this constant before.